gradle/libs.versions.toml

```diff
@@ -71,6 +75,7 @@ coroutines-android = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutin
 
 # Core
 koin = { group = "io.insert-koin", name = "koin-android", version.ref = "koin" }
+koin-compose = { group = "io.insert-koin", name = "koin-androidx-compose", version = "3.2.0" }
```

Comment:
Add a [versions] name

Reply:
Good catch. It should actually just use the default koin version, but the koin maintainers forgot to push this artifact for the latest version, thus I had to downgrade..

Reply:
The compose part is in a separate repository (the normal androidx libraries are in a monorepo with core) so I think splitting into two versions is the way to go, worst case we bump two versions at the same time (although Renovate helps with that).